pkg/services/featuremgmt/registry.go

```diff
@@ -224,15 +224,15 @@ var (
 			Name:              "unifiedRequestLog",
 			Description:       "Writes error logs to the request logger",
 			Stage:             FeatureStageGeneralAvailability,
-			Owner:             grafanaBackendGroup,
+			Owner:             grafanaBackendServicesSquad,
 			Expression:        "true",
 			HideFromAdminPage: true,
 		},
 		{
 			Name:              "renderAuthJWT",
 			Description:       "Uses JWT-based auth for rendering instead of relying on remote cache",
 			Stage:             FeatureStagePublicPreview,
-			Owner:             grafanaAsCodeSquad,
+			Owner:             grafanaSharingSquad,
```

Comment:
@nmarrs  ptal

Reply:
@AgnesToulet does this change seem reasonable? If yes please feel to approve PR of behalf of sharing :)

Reply:
Yes, it will probably change again soon but sounds good for now :+1: